sdk/core/core-http/src/fetchHttpClient.ts

```diff
@@ -18,14 +18,23 @@ interface FetchError extends Error {
   type?: string;
 }
 
+/**
+ * String URLs.
```

Comment:
Why is it called “CommonRequestInfo” then? The name seems too complicated to be a string URL.